var/spack/repos/builtin/packages/wxparaver/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Wxparaver(AutotoolsPackage):
+    """"A very powerful performance visualization and analysis tool
+        based on traces that can be used to analyse any information that
+        is expressed on its input trace format.  Traces for parallel MPI,
+        OpenMP and other programs can be genereated with Extrae."""
+    homepage = "https://tools.bsc.es/paraver"
+    url = "https://ftp.tools.bsc.es/wxparaver/wxparaver-4.6.3-src.tar.bz2"
```

Comment:
```suggestion
    url = "https://ftp.tools.bsc.es/wxparaver/wxparaver-4.9.2-src.tar.bz2"
```
Downloads will still work regardless, but our url parsing tests assume the version is defined in the package